src/ui/Crosshair.js

```diff
@@ -8,8 +8,44 @@ export default class Crosshair extends LineSegments {
     const geometry = createGeometry(0.05);
     const material = createMaterial();
 
-    super(geometry, material);
+    const obj = super(geometry, material);
+
+    this.size = 0.05;
+    this.selected = false;
+    this.timer = 0;
+    this.timeLimit = 30;
+    return obj;
+  }
+
+  update(selector) {
+    // TODO: Rewrite this animation using tweens.
```

Comment:
This works for now but in the future we should use tweens for these kinds of animations. I added a todo comment to remind us.